api/desecapi/serializers.py

```diff
@@ -760,7 +760,7 @@ def to_representation(self, instance: models.AuthenticatedAction):
 
     def to_internal_value(self, data):
         # calculate code TTL
-        validity_period = self.context.get('validity_period', self.validity_period)
+        validity_period = self.context.get('validity_period', self.validity_period)  # TODO understand why necessary
```

Comment:
to make sure that serializer in authentication.py gets the right validity period

Improve comment; also, try if fallback is needed or subscript suffices

Reply:
done